src/System.Management.Automation/engine/hostifaces/PSCommand.cs

```diff
@@ -85,19 +85,19 @@ internal PSCommand(Command command)
         /// <exception cref="ArgumentNullException">
         /// cmdlet is null.
         /// </exception>
-        public PSCommand AddCommand(string command)
+        public PSCommand AddCommand(string cmdlet)
         {
-            if (command == null)
+            if (cmdlet == null)
             {
-                throw PSTraceSource.NewArgumentNullException("cmdlet");
```

Comment:
@rjmholt I think there is a bug here where a potential null argument is referred to using string "cmdlet". The actual argument name is not "cmdlet" but rather "command". If you agree, perhaps I can replace this line with:

`throw PSTraceSource.NewArgumentNullException(nameof(command));`

Reply:
Please pull new PR with the change.

Reply:
> Please pull new PR with the change.

Just would like some clarification -- should I create a new GitHub issue and then create a new PR linked to that issue?

Reply:
Issues are need only for discussions to understand what we should do or not do. Here we see simple inconsistency and can accept new PR.

Reply:
> Issues are need only for discussions to understand what we should do or not do. Here we see simple inconsistency and can accept new PR.

Got it. Thank you.